events/events.go

```diff
@@ -116,7 +119,12 @@ func writeEvent(job *engine.Job, event *utils.JSONMessage, eventFilters filters.
 		return true
 	}
 
-	if isFiltered(event.Status, eventFilters["event"]) || isFiltered(event.From, eventFilters["image"]) || isFiltered(event.ID, eventFilters["container"]) {
+	for i, cn := range eventFilters["container"] {
+		eventFilters["container"][i] = GetContainerId(job.Eng, cn)
```

Comment:
@brahmaroutu Can you add a comment explaining why we are overwriting the `container` filter?